### ISSUE TITLE
Problem running test in Jenkins CI

### ISSUE THREAD

**Issue by pveyes** [DevExpress/testcafe-browser-provider-browserstack#14]
Hi I'm having problem with running testcafe in our Jenkins CI. It stuck and doesn't exit properly after running the tests. The test itself is triggered and completed (I can see them in Browserstack dashboard), but the process in Jenkins doesn't stopped. This doesn't happen locally.

Here's a timeline of our Jenkins build

```
10:43:47 $ testcafe "browserstack:Chrome@53.0:Windows 10" suite/mobile/home/MobileHomePage.js 
10:44:55  Running tests in:
10:44:55  - Chrome 53.0.2785 / Windows 10 0.0.0
10:44:55 
10:44:55  Mobile Home page
10:44:55  ✓ Usable menu
10:44:55 
10:44:55 
10:44:55  1 passed (8s)
Aborted by Fatih Kalifa
10:51:47 Sending interrupt signal to process
10:51:57 After 10s process did not stop
```

I also notice there's a slight delay locally after printing `1 passed` and exiting, and I think this is where the cleanup phase happen (stopping browserstack worker?)

If you have suggestion on how to debug this I would gladly appreciated

**Comment by pveyes** [comment]
Not sure if useful but this is my test suite

```
import { Selector } from 'testcafe';

fixture('Mobile Home page')
  .page('https://m-ui-00.test.traveloka.com');

const menu = Selector('[data-url="#menu"] svg');
const menuLink = Selector('[class*="HybridLink"] div');

test('Usable menu', async t => {
  await t.click(menu);
  await t.click(menuLink);
});
```

**Comment by AndreyBelym** [comment]
Hello @pveyes, I assume it happens because [browserstack-local binary](https://github.com/browserstack/browserstack-local-nodejs) is unable to close a tunnel to Browserstack. I will upload debug build to the `npm` shortly, so can you try it and check if it will work on Jenkins?

**Comment by AndreyBelym** [comment]
You can install the debug build with:
`npm install --save testcafe-browser-provider-browserstack@1.1.1-git.201706070945`

**Comment by pveyes** [comment]
Hi @AndreyBelym it works now, thanks 🙇

**Comment by dgieselaar** [comment]
Hi @AndreyBelym, this is indeed fixed in `1.1.1-git.201706070945`, but for `1.4.0`, we still have this issue (indeed in Jenkins, with Browserstack). Any idea as to why this happens?
